lua/lazyvim/plugins/extras/lang/typst.lua

```diff
@@ -0,0 +1,36 @@
+return {
+  recommended = function()
+    return LazyVim.extras.wants({
+      ft = { "typst" },
+    })
+  end,
+  {
+    "nvim-treesitter/nvim-treesitter",
+    opts = { ensure_installed = { "typst" } },
+  },
+  {
+    "neovim/nvim-lspconfig",
+    opts = {
+      servers = {
+        typst_lsp = {},
+      },
+    },
+  },
+  -- Typst preview
+  {
+    "chomosuke/typst-preview.nvim",
+    ft = "typst",
+    version = "0.3.*",
```

Comment:
This will lock the plugin version to `0.3.*`. If there are any updates to the plugin version it won't get updated. Any reason this is needed? All plugins in LazyVim (except if I'm forgetting something) install the latest HEAD.

Reply:
Following the instructions in https://github.com/chomosuke/typst-preview.nvim

Reply:
But please, I am open to suggestions

Reply:
My suggestion as just a simple user would be to not include `version` at all.

Reply:
Done in 2c3103a04aabaf316fcb80e01a03baddcbb5e35e